tests/framework/extensions/provisioning/verify.go

```diff
@@ -96,8 +96,10 @@ func VerifyRKE1Cluster(t *testing.T, client *rancher.Client, clustersConfig *clu
 		}
 	}
 
-	podErrors := pods.StatusPods(client, cluster.ID)
-	assert.Empty(t, podErrors)
+	if !strings.Contains(clustersConfig.CloudProvider, "external") {
```

Comment:
We don't want to do a pod status check if it is an outside cloud provider?

Reply:
validation for external cloud providers will be separate, since all the nodes are tainted at this stage. there will be some pods in an error state due to the taints, so I special cased this in the validation, and re-check the pods after the chart and its resources have deployed successfully.